scheduler.go

```diff
@@ -262,7 +274,9 @@ func (s *Scheduler) Shutdown() {
 
 	s.clearHistory()
 	s.client.Close()
```

Comment:
In the case of `scheduler.Shutdown`, the client is closed by Asynq. If it uses a shared pool, it will return an error (which is unhandled here). Perhaps it should be moved to the following `if` clause?

If the scheduler ends up using a shared redis connection pool, perhaps it should only be stopped. Right now there doesn't seem to be an option to simply Stop/Pause the scheduler though